iframe/src/mocks/responses/gateways.ts

```diff
@@ -1393,18 +3100,169 @@ const gatewaysAllParams: GatewaysResponse = {
+        },
```

Comment:
I converted the icons of the new payment methods from https://xanpool.com/ to a base64 representation and included them here.